lib/paperclip/paperclip_processors/transcoder.rb

```diff
@@ -56,6 +56,18 @@ def make
         if output_is_image?
           @time = @time.call(@meta, @options) if @time.respond_to?(:call)
           @cli.filter_seek @time
+
+          if @auto_rotate && !@meta[:rotate].nil?
+            log "Adding rotation #{@meta[:rotate]}"
+            case @meta[:rotate]
+            when 90
+              @convert_options[:output][:vf] = "'transpose=1'"
```

Comment:
`av` gem supports rotation using `filter_rotate`. And the interface is the same for `avconv` and `ffmpeg`. So maybe we should keep this `auto_rotate` option in this gem but call `filter_rotate` instead.

In the meantime I tested this and it's not working properly because arguments are not being passed correctly. I'm opening another PR in `av` gem to fix this: https://github.com/ruby-av/av/pull/15